zap/src/main/java/org/zaproxy/zap/extension/api/CoreAPI.java

```diff
@@ -1294,6 +1317,17 @@ public void process(RecordHistory recordHistory) {
             result =
                     new ApiResponseElement(
                             name, getAlertParam(ApiException.Type.BAD_VIEW).getOverridesFilename());
+        } else if (VIEW_LOGGING.equals(name)) {
+            String logName = ApiUtils.getOptionalStringParam(params, PARAM_NAME);
```

Comment:
`getParam(params, PARAM_NAME, "")`, otherwise the `logName` might be null and lead to NPE.